### PR TITLE
Feat : Add Model Component

### PR DESCRIPTION
**Pull request by SaurabhChawda**
[Modal 1](https://user-images.githubusercontent.com/90862803/152485984-5238d187-c689-4ad4-9ccc-06f583108257.png)
Modal 2!

Added Modal Component, In which I have added a button and a pop-up message Container.

### CODE REVIEW

File: assets/Css/ComponentsStyles.css [hunk 1]
```diff
@@ -426,4 +426,56 @@ main {
...
+  align-items: center;
+  z-index: 1;
+  position: fixed;
+  top:50%;
+  left:50%;
+  transform: translate(-50%,-50%);
+  border:2px solid blueviolet;
+  color: black;
+  background-color: white;
+  box-shadow: 0px 0px 5px grey;
```
Comment (t007rushi) [inline assets/Css/ComponentsStyles.css:477]: Use variables for colors

**Review by t007rushi — CHANGES_REQUESTED** [review]
Small changes only to push

File: assets/Css/ComponentsStyles.css (hunk 1 above)
Comment (SaurabhChawda) [reply assets/Css/ComponentsStyles.css:477]: Yes but this is a component Color that why I have used direct color. Should I use color var for this?

**Review by ShreyashMahajan — APPROVED** [review]
code looks clean. Good to merge

**Review by nikhilmalviya-dev — APPROVED** [review]
great work

**Comment by SaurabhChawda** [comment]
> code looks clean. Good to merge

Thankyou

**Comment by SaurabhChawda** [comment]
> great work

ThankYou

**Pull request merged**